server/monitor.go

```diff
@@ -370,6 +426,26 @@ func decodeInt(w http.ResponseWriter, r *http.Request, param string) (int, error
 	return val, nil
 }
 
+func decodeState(w http.ResponseWriter, r *http.Request) (ConnState, error) {
+	str := r.URL.Query().Get("state")
+	if str == "" {
+		return ConnOpen, nil
+	}
+	switch strings.ToLower(str) {
+	case "open":
```

Comment:
should this be `opened` since you have `closed` (or add both open, opened and close, closed.

Reply:
ok